src/js/components/DataTable/DataTable.js

```diff
@@ -388,7 +388,11 @@ const DataTable = ({
 
   const Container = paginate ? StyledContainer : Fragment;
   const containterProps = paginate
-    ? { ...theme.dataTable.container, fill }
+    ? {
+        ...theme.dataTable.container,
+        fill,
+        ...{ passThemeFlag },
```

Comment:
```suggestion
        ...passThemeFlag,
```
Can it just be this?

Reply:
Good note, fixed in latest commit